example_client/east_ocr_client.py

```diff
@@ -16,6 +16,7 @@
 
 import grpc
 import cv2
+import os
```

Comment:
Left by mistake?

Reply:
Previous commit had it missing. So without this line currently develop branch has broken east ocr client.